build-system/common/update-session-issues/template/design-review.js

```diff
@@ -6,12 +6,18 @@ const sessionDurationHours = 1;
  * also be updated correspondingly.
  */
 const timeRotationUtc = [
+  [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
+  [/* wed */ 3, '21:00', 'Americas'],
+  [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
+  [/* wed */ 3, '21:00', 'Americas'],
+  [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
+  [/* wed */ 3, '21:00', 'Americas'],
   [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
   [/* wed */ 3, '21:00', 'Americas'],
   [/* thu */ 4, '01:00', 'Asia/Oceania'],
```

Comment:
Does this line need to be removed?

Reply:
I don't think so. The [first step](https://github.com/ampproject/amphtml/issues/36888#issuecomment-983843325) that came out of design review was simply a reduced cadence.

Reply:
Like @samouri said, this changes cadence.